gapis/api/commandGenerator/linear_command_generator.go

```diff
@@ -29,8 +29,18 @@ type linearCommandGenerator struct {
 // takes a list of cmds and returns them in order when GetNextCommand
 // is called.
 func NewLinearCommandGenerator(initialCommands []api.Cmd, realCommands []api.Cmd) CommandGenerator {
+	commands := make([]api.Cmd, 0)
```

Comment:
Possibly stupid question: what is the effect of the changes in this function? Was the previous implementation buggy? Or you're trying to make the code more explicit?

Reply:
To make it more explicit but I can revert if you think this is redundant.